client/app/components/pages/GeneHome.vue

```diff
@@ -2805,6 +2806,12 @@ export default {
           }
         }
 
+        let isTrio = self.paramRelationships.every(sample => sample);
+        if (isTrio) {
+          self.isMother = true;
+          self.isFather = true;
+        }
```

Comment:
If the gene is launched from the URL this checks if the `paramRelationships` array has the value "mother" and "father" in the array or if it is undefined. If it is true, it sets `isMother` and `isFather` as true. 

`paramRelationships` is set from the route parameters and is an array with 3 indexes. If it is a singleton, the last 2 indexes are set as undefined.